api.py

```diff
@@ -39,37 +30,24 @@ def handle_validation_error(e):
         'errors': [validation_error.message for validation_error  in e.errors],
     }), 400
 
+@api.before_request
+def before_request():
+    ensure_datadir()
```

Comment:
On every request check that data directory is in order. This is just a simple way to make sure that it's always there. Not efficient but ensures that it's always there.